README.rst

```diff
@@ -93,3 +93,30 @@ In order to activate these processor some configuratios are required.
 
 
 .. _ecommerce: https://github.com/edx/ecommerce/
+
+Releases
+########
+Before performing a release, it is important to take into account the following:
+
+- A PR of the branch containing the changes to 'main' is created.
+- The branch must be up to date with respect to 'main'. Use `rebase`_ for this.
+- The PR must have an update of the file `CHANGELOG.rst`_. All changes contained in the branch to be merged must be specified as 'unreleased'.
+- Once the PR is approved, we perform squash and merge.
+
+To release this repository:
+
+1. In development, we position ourselves in the main branch and update it. 
+2. `bumpversion`_: Then we write in console::
+
+    bumpversion [major, minor, patch]
+3. The changes that are in 'unreleased' become 'released', they are identified with the date and the version that has generated `bumpversion`_.
+4. The commit generated by `bumpversion`_ is pushed, which must also include the recent change in `CHANGELOG.rst`_.
```

Comment:
what about moving the changes from 'unreleased'?

Reply:
You mean when is the commit done? As I see it, bumpversion is done, then 'unreleased' is moved to 'released', and that commit is squashed with the bumpversion to upload a single commit. 

@andrey-canon Can you confirm if this is so?

Reply:
I think the `that commit is squashed with the bumpversion to upload a single commit.` is not necessary

Reply:
@mariajgrimaldi I understand. You are probably correct. But how do you suggest these kinds of confirmations be made? [https://github.com/eduNEXT/eox-audit-model/commit/201a982f2d1907c62daed0377e4869ce8eb04797](https://github.com/eduNEXT/eox-audit-model/commit/201a982f2d1907c62daed0377e4869ce8eb04797)

Reply:
Great. Then what you said is fine